client/pkg/logutil/zap.go

```diff
@@ -33,6 +34,39 @@ func CreateDefaultZapLogger(level zapcore.Level) (*zap.Logger, error) {
 	return c, nil
 }
 
+// CreateUtilZapLogger creates a logger with default zap configuration can redirect log to /dev/null
+func CreateUtilZapLogger(level zapcore.Level) *zap.Logger {
+	lcfg := DefaultZapLoggerConfig
+	lcfg.Level = zap.NewAtomicLevelAt(level)
+	lcfg.Development = false
+	lcfg.Encoding = DefaultLogFormat
+	lcfg.Sampling = &zap.SamplingConfig{
+		Initial:    100,
+		Thereafter: 100,
+	}
+	infoLevel := zap.LevelEnablerFunc(func(level zapcore.Level) bool {
+		return level == zapcore.InfoLevel
+	})
+	errorFatalLevel := zap.LevelEnablerFunc(func(level zapcore.Level) bool {
+		return level == zapcore.ErrorLevel || level == zapcore.FatalLevel
```

Comment:
The problem with this is that it's not handling the `WarningLevel`, nor `PanicLevel`. It is an enumeration, so it would be useful to compare using greater than.